crates/supervisor/core/src/reorg/handler.rs

```diff
@@ -56,19 +56,23 @@ where
                 managed_node.clone(),
             );
 
-            let handle = tokio::spawn(async move { reorg_task.process_chain_reorg().await });
+            let chain_id = *chain_id;
+
+            let handle = tokio::spawn(async move {
+                let reorg_task = reorg_task.with_metrics();
+                let start_time = Instant::now();
+                let result = reorg_task.process_chain_reorg().await;
+                super::metrics::Metrics::record_l1_reorg_processing(chain_id, start_time, &result);
```

Comment:
```suggestion
                Metrics::record_l1_reorg_processing(chain_id, start_time, &result);
```
just bring `Metrics` into scope in the module prelude